src/main.c

```diff
@@ -1052,7 +1052,22 @@ static void brake(data *d) {
     }
 
     VESC_IF->timeout_reset();
-    VESC_IF->mc_set_brake_current(d->float_conf.brake_current);
+
+    // If brake current is set to 0 don't do anything
+    if (d->float_conf.brake_current == 0) {
+        return;
+    }
+
+    // Use brake current over certain ERPM to prevent the board skidding to a stop when deactivated
+    // at speed?
```

Comment:
Is the question mark in the comment intentional? It doesn't make me feel very confident :slightly_smiling_face: